daml-lf/transaction-scalacheck/src/main/scala/com/digitalasset/daml/lf/value/ValueGenerators.scala

```diff
@@ -154,7 +154,9 @@ object ValueGenerators {
   val absCoidV0Gen: Gen[AbsoluteContractId.V0] =
     Gen.alphaStr.map(t => Value.AbsoluteContractId.V0.assertFromString('#' +: t))
   private val genAbsCidV1: Gen[AbsoluteContractId.V1] =
-    Gen.zip(genHash, genBytes) map { case (h, b) => AbsoluteContractId.V1.assertBuild(h, b) }
+    Gen.zip(genHash, genBytes.filter(_.length <= 94)) map {
```

Comment:
Where did this `94` come from? Might be worth pulling it out into a constant and explaining it. Is it just to make sure the generator keeps things small enough?

Reply:
Avoid `filter` unless the price of doing without is very high. (That's why the other code below enforces "non-empty suffix" via "generate another byte and append it so there must have been at least 1" rather than "filter out empty suffixes".)

In this case, I would rename `genBytes` to `genBytes94` and define it as

```scala
  for
    sz <- Gen.chooseNum(0, 94)
    ab <- Gen.containerOfN[Array, Byte](sz, arbitrary[Byte])
  } yield Bytes fromByteArray ab
```

Reply:
Done. Thanks.